sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -1,5 +1,20 @@
 namespace Azure
 {
+    public partial class CloudMachine
+    {
+        public CloudMachine(Microsoft.Extensions.Configuration.IConfiguration configuration) { }
```

Comment:
Is IConfiguration a type we plan to leverage elsewhere?

Reply:
We started to talk about adding this as an allowed dependency and using it in Azure.Core. For example, we wanted to use it in a new cloud configuration type that @johanste, @schaabs, and @m-nash are experimenting with.

Reply:
Though, this is an outdated comment. I removed the API for now.